AstSemantics.md

```diff
@@ -227,6 +227,11 @@ MVP, there are [future features](FutureFeatures.md#finer-grained-control-over-me
 proposed to allow setting protection and creating mappings within the
 contiguous linear memory.
 
+In the MVP, the result type of `page_size` is int32`; in the future, support
+support for page sizes requiring a type of `int64` may be added. The page size
+is an unsigned integer which is a power of 2 and at least 1024. `page_size`
```

Comment:
Mention that this is bytes?
Also, something along the lines of "it's not necessarily the actual page size, it's a suggested assumption from the runtime".